5-flexbox/style.css

```diff
@@ -0,0 +1,54 @@
+body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
+    font-family: 'Balsamiq Sans';
+    background-color: #E8E8E8;
+}
+
+h1 {
+    text-align: center;
+    font-size: 40px;
+    margin-top: 50px;
+    font-weight: 100;
+    font-style: italic;
+}
+
+main {
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: auto;
```

Comment:
it's more common to do: `margin: 0 auto;` due the elements stack vertically by default.